### ISSUE TITLE
Kubric dataset

### ISSUE THREAD

**Issue by XiongyiCai** [deepmind/tapnet#18]
Hi, when I use the Kubric(Movi-E), I find the number of the train set is about 9750, the validation set is about 250 and the test set is about 999, which is different with the number 38,325/799 for train/validation in paper.

**Comment by cdoersch** [comment]
It's unlikely that Klaus or Andrea will see this comment here, and they're the ones that generated the external MOVi-E.  As I suggested over email, the Kubric repository is probably a better place for this issue.
